.travis.yml

```diff
@@ -3,23 +3,17 @@ os:
   - linux
   - osx
 julia:
-  - 0.7
   - 1.0
-  - 1.1
-  - 1.2
+  - 1.4
```

Comment:
This could just be `1` so that the latest stable release gets run.

Reply:
Nice! 👍🏻